release/src/main/scripts/build_release_candidate.sh

```diff
@@ -228,20 +229,14 @@ if [[ $confirmation = "y" ]]; then
   cd ~/${LOCAL_WEBSITE_UPDATE_DIR}/${LOCAL_WEBSITE_REPO}
   git clone ${GIT_BEAM_WEBSITE}
   cd ${WEBSITE_ROOT_DIR}
-  git checkout -b updates_release_${RELEASE}
-
-  echo "..........Updating release version in _config.yml..........."
-  sed -i -e "s/release_latest: [0-9].[0-9].[0-9]/release_latest: ${RELEASE}/g" _config.yml
+  git checkout release-docs
+  git checkout -b updates_release_${RELEASE} release-docs
 
   echo "..........Copying generated javadoc into beam-site.........."
-  cp -r ${GENERATE_JAVADOC} src/documentation/sdks/javadoc/${RELEASE}
-  echo "Updating javadoc current.md with latest release version number"
-  sed -i -e "s/[0-9].[0-9].[0-9]/${RELEASE}/g" src/documentation/sdks/javadoc/current.md
+  cp -r ${GENERATE_JAVADOC} javadoc/${RELEASE}
 
   echo "............Copying generated pydoc into beam-site.........."
-  cp -r ${GENERATED_PYDOC} src/documentation/sdks/pydoc/${RELEASE}
-  echo ".....Updating pydoc current.md with latest release version number......"
-  sed -i -e "s/[0-9].[0-9].[0-9]/${RELEASE}/g" src/documentation/sdks/pydoc/current.md
+  cp -r ${GENERATED_PYDOC} pydoc/${RELEASE}
 
   git add -A
   git commit -m "Update beam-site for release ${RELEASE}"
```

Comment:
It would be useful to have the git tag there were generated from in the commit message.

You can grab the commit from the release branch after cloning the apache/beam repo (line 90):

```
  RELEASE_COMMIT=$(git rev-parse --verify ${RELEASE_BRANCH})
```

and then.. 

```suggestion
  git commit -m "Update beam-site for release ${RELEASE}\n\nContent generated based on commit ${RELEASE_COMMIT}"
```

Reply:
Done